starfish_py/__init__.py

```diff
@@ -1,16 +1,27 @@
 """
     Starfish-py module
 
-    The main class to call is the :func:`starfish_py.ocean.Ocean` class.
-
-
+    This module contains the following classes
+
+    * :class:`Account` The Account class.
+    * :class:`Agent` The Agent wrapper class.
+    * :class:`Asset` The Asset class.
+    * :class:`Config` The Config data class.
+    * :class:`Ocean` The main Ocean connection class.
+    * :data:`logger` The logger class.
 """
 
-
 __author__ = """DEX.sg"""
 __version__ = '0.0.1'
 
-
 import logging
 
 logger = logging.getLogger(__name__)
+
+
+from .account import Account
+from .agent import Agent
+from .asset.asset import Asset
+from .asset.asset_light import AssetLight
```

Comment:
Is _asset_light_ just a temporary import? By it's name it sounds like it does less that _asset_ which is imported just before it.